src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -462,6 +535,7 @@ public int onBite(IVampire biter) {
         int sucked = (int) (amt * perc);
         bloodStats.removeBlood(sucked, true);
         sync(this.bloodStats.writeUpdate(new NBTTagCompound()), true);
+        feedCounter = 30;
```

Comment:
Is this going to be used somewhere?

Reply:
oh I added it as part of the overlay code and forgot to remove it.